src/corva/service/service.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Callable
+from unittest import mock
+
+from corva import shared
+from corva.service.api_sdk import ApiSdkProtocol
+
+
+def run_app(
+    has_secrets: bool,
+    app_key: str,
+    api_sdk: ApiSdkProtocol,
+    app: Callable[[], Any],
+) -> Any:
+    secrets = api_sdk.get_secrets(app_key=app_key) if has_secrets else {}
```

Comment:
why do we need a `get_secrets` param if we can try to get secrets right away and fill it with empty dict if none found?

Reply:
`get_secrets` is skipped if `has_secrets is False`.

`print(1) if False else print(2)` prints only 2

Reply:
https://github.com/corva-ai/python-sdk/pull/56/commits/7c9413d47b00fdb2cc53c3b8b68b897cea08cac2